bitrise.yml

```
@@ -16,10 +16,23 @@ workflows:
     before_run:
       - setup
     steps:
+      - script@1:
+          inputs:
+            - content: |-
+                #!/usr/bin/env bash
+                envman add --key YARN_CACHE_DIR --value "$(yarn cache dir)"
+          title: Get Yarn cache directory
+      - cache-pull@2: {}
       - yarn@0:
           inputs:
             - command: setup
           title: Yarn Setup
+      - cache-push@2:
+          # Manual cache path needed here because the `yarn` step only caches for the `install` command
+          inputs:
+            # The Yarn cache is easier and safer to cache than `node_modules`, which could include
+            # postinstall script modifications and could affect hoisting on subsequent installs.
+            - cache_paths: "$YARN_CACHE_DIR"
```

Do we have to use the `yarn.lock` as an "update indicator file" here?

---

Ah, good catch, that does seem like the right thing to do here.

```suggestion
            - cache_paths: "$YARN_CACHE_DIR -> ./yarn.lock"
```